option optimize_for = LITE_RUNTIME;

message HeartbeatRequest
{
 required uint32 times = 1;
}
